src/quickstart/deploy-an-app.md

```diff
@@ -95,14 +95,20 @@ proper-barbecue-v18-cmd-rk644 up (v18)
 proper-barbecue-v18-cmd-0ag04 up (v18)
 ```
 
-Congratulations! You have deployed, configured, and scaled your first application using Deis Workflow. There is a lot
-more you can do with Deis Workflow, play around with the CLI:
+Congratulations! You have deployed, configured, and scaled your first application using Deis Workflow. 
+
+## Going Further
+There is a lot more you can do with Deis Workflow, play around with the CLI:
+
+> **IMPORTANT:** In order to have permission to push an app you must add a SSH key
+> to your user on the Deis Workflow. For more information, please check
+> [Users and SSH Keys](../users/ssh-keys/) and [Troubleshooting Workflow](../troubleshooting/) sections out.
```

Comment:
I wonder if this deserves a totally different section or should go into the sections about using buildpacks and/or dockerfiles.